src/NationalInstruments.Analyzers/Style/ChainOfMethodsWithLambdasAnalyzer.cs

```diff
@@ -98,14 +125,18 @@ private static void AnalyzeSyntaxNode(SyntaxNodeAnalysisContext context)
                 if (!dotToken.LeadingTrivia.Any(trivia => trivia.IsKind(SyntaxKind.WhitespaceTrivia)))
                 {
                     var diagnostic = Diagnostic.Create(Rule, invocationExpressionSyntax.GetLocation());
-                    context.ReportDiagnostic(diagnostic);
+                    reportDiagnostic(diagnostic);
                     break;
                 }
             }
 
             bool IsLambdaExpression(SyntaxNode syntaxNode) => syntaxNode.IsKind(SyntaxKind.SimpleLambdaExpression);
 
+            bool IsNonNested(SyntaxNode syntaxNode) => IsNotArgumentSyntax(syntaxNode) && IsNotArrayInitializerSyntax(syntaxNode);
```

Comment:
I don't know what "non-nested" means based on this definition. Is there another name more descriptive?

Reply:
How about [this](https://github.com/ni/csharp-styleguide/pull/65/commits/96736965f3e7d9fa4377598ba4740bf24350ef64)?